src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -205,4 +213,151 @@ public boolean isSelected() {
+        if (!"POST".equals(req.getMethod())) {
+            rsp.sendRedirect2(".");
+            return;
+        }
+        JSONObject json = req.getSubmittedForm();
+        if (!json.has("components")) {
+            rsp.sendError(HttpServletResponse.SC_BAD_REQUEST);
+            return;
+        }
+
+        List<Integer> indexList = getSelectedIndices(req);
+        BundleBrowser bundleBrowser = BundleBrowser.getBundleBrowser();
+        bundleBrowser.deleteBundle(indexList);
+        PrintWriter out = rsp.getWriter();
+        out.println("</script>");
+        rsp.forwardToPreviousPage(req);
+
+        rsp.setContentType("text/html");
+        out.println("<script type=\"text/javascript\">");
+        out.println("alert('Bundle(s) Deleted!');");
+        try {
+            SupportPlugin.setRequesterAuthentication(Jenkins.getAuthentication());
+            SecurityContext old = ACL.impersonate(ACL.SYSTEM);
+            SecurityContextHolder.setContext(old);
+            SupportPlugin.clearRequesterAuthentication();
+        } finally {
+            logger.fine("Response completed");
+        }
+    }
+
+    private List<Integer> getSelectedIndices(StaplerRequest req) throws ServletException {
+        ArrayList<Integer> selectedFileIndices = new ArrayList<Integer>();
+        JSONObject json = req.getSubmittedForm();
+        try{
+            req.getSubmittedForm().getJSONObject("components");
+            selectedFileIndices.add(0);
+        }catch (Exception e) {
+            JSONArray jsonArray = json.getJSONArray("components");
+
+            for (int i = 0; i < jsonArray.size(); i++) {
+                String isSelected = jsonArray.getJSONObject(i).get("selected").toString();
+                if ("true".equals(isSelected)) {
+                    selectedFileIndices.add(i);
+                }
+            }
+        }
+        return selectedFileIndices;
+    }
+
+    @Restricted(NoExternalUse.class)
+    public void doConfigure(StaplerRequest req,StaplerResponse rsp) throws ServletException, IOException {
+        String fileName = "config.txt";
+        String numberOfDays = req.getSubmittedForm().getString("numberOfDays");
+        try {
+            Integer.parseInt(numberOfDays);
+            try {
+                File file = new File(fileName);
+                FileWriter fileWriter = new FileWriter(file);
+                fileWriter.append(numberOfDays);
+                fileWriter.close();
+                rsp.forwardToPreviousPage(req);
+            } catch (FileNotFoundException ex) {
+                logger.log(Level.FINE, ex.getMessage(), ex);
+            } catch (IOException ex) {
+                logger.log(Level.FINE, ex.getMessage(), ex);
+            }
+        }catch (Exception e) {
+            PrintWriter out = rsp.getWriter();
+            rsp.setContentType("text/html");
+            out.println("<script type=\"text/javascript\">");
+            out.println("alert('Input must be an integer!');");
```

Comment:
This should be instead replaced with a `doCheck` method as we're only checking for the version number (see https://youtu.be/eUzYZZsNBIA?t=37m24s)